test/specs/scenarios/accessibility.js

```diff
@@ -0,0 +1,186 @@
+import { runAccessibilityTest } from '../../utils/accessibility'
+
+const options = {
+  pageObjects: [
+    'Welcome',
+    'Confirm',
+    'DocumentSelector',
+    'DocumentUpload',
+    'CrossDeviceIntro',
+    'CrossDeviceLink',
+    'CrossDeviceMobileConnected',
+    'CrossDeviceSubmit',
+    'Camera',
+    'LivenessIntro',
+    'PoaDocumentSelection',
+    'PoaGuidance',
+    'PoaIntro',
+    'VerificationComplete',
+    'BasePage'
+  ]
+}
+
+export const accessibilityScenarios = async(lang='en') => {
+
+  describe(`ACCESSIBILITY scenarios in ${lang}`, options, ({driver, pageObjects}) => {
+
+    const {
+      welcome,
+      confirm,
+      documentSelector,
+      documentUpload,
+      crossDeviceIntro,
+      livenessIntro,
+      poaDocumentSelection,
+      poaIntro,
+      basePage
+    } = pageObjects
+
+    const copy = basePage.copy(lang)
+
+    const goToPoADocumentSelectionScreen = async () => {
+      driver.get(localhostUrl + `?poa=true&async=false&useWebcam=false`)
+      welcome.primaryBtn.click()
+      poaIntro.clickStartVerificationButton()
+    }
+
+    const goToCrossDeviceScreen = async () => {
+      welcome.primaryBtn.click()
+      documentSelector.passportIcon.click()
+      documentUpload.crossDeviceIcon.click()
+      crossDeviceIntro.continueButton.click()
+    }
+    
+    //Welcome
+    it('should verify accessibility for the welcome screen', async () => {
+      driver.get(`${localhostUrl}?language=${lang}`)
+      runAccessibilityTest(driver)
+    })
+
+    it('should verify focus management for the welcome screen', async () => {
+      driver.get(`${localhostUrl}?language=${lang}`)
+      welcome.verifyFocusManagement()
+    })
+
+    //Cross Device Sync
+    it('should verify accessibility for the cross device intro screen', async () => {
+      driver.get(`${localhostUrl}?language=${lang}`)
+      welcome.primaryBtn.click()
+      documentSelector.passportIcon.click()
+      documentUpload.crossDeviceIcon.click()
+      runAccessibilityTest(driver)
+    })
+
+    it('should verify accessibility for the cross device screen', async () => {
+      driver.get(`${localhostUrl}?language=${lang}`)
+      goToCrossDeviceScreen()
+      runAccessibilityTest(driver)
+    })
+
+    // Document Selector
+    it('should verify accessibility for the document selector screen', async () => {
+      driver.get(`${localhostUrl}?language=${lang}`)
+      welcome.primaryBtn.click()
+      runAccessibilityTest(driver)
+    })
+
+    //Document Upload
+    it('should verify accessibility for the uploader screen', async () => {
+      goToPassportUploadScreen(driver, welcome, documentSelector, `?language=${lang}`)
+      runAccessibilityTest(driver)
+    })
+
+    //FIXME: This is commented out due to the accessibility violations we need to fix.
+    // it('should verify accessibility for the document upload confirmation screen', async () => {
```

Comment:
This one is returning a valid error that should be fixed. The `aria-live` attribute in the document previewer does not have any value.  In `src/components/EnlargedPreview/index.js` you can see that aria live has an empty string in some cases `aria-live={isExpanded ? 'assertive' : ''}` which results in the following code.
![Screen Shot 2019-10-08 at 11 09 34](https://user-images.githubusercontent.com/7127427/66387829-aef62080-e9bc-11e9-8681-70fbc4260fa7.png)
Try using `aria-live={isExpanded ? 'assertive' : 'off'}` ( I remember that using `undefined` was not working as expected) and see if that maintain the behaviour and fixes the breaking test.

Reply:
I tried that and it fixed the test ✨ . I also checked VoiceOver and Talkback behavior of the document upload confirmation screen and some others and didn't notice any wrong behavior.

Reply:
Great. Did you test it on mobile devices too?

Reply:
Yes. On a desktop with VoiceOver and on mobiles with VoiceOver and Talkback

Reply:
Is there a way to test for NVDA or JAWS on Windows (IE/Edge)?

Reply:
🤔 IDK, would need to explore more outside of that ticket probably.

Reply:
No worries. Just a thought. :)